### PR TITLE
Get Started organization updates

### PR DESCRIPTION
**Pull request by mDuo13**
- Split off "Public Servers" page
- Rename and reorder getting started pages by language
- Update links to xrpl4j in client libraries page

### CODE REVIEW

File: content/tutorials/get-started/get-started-using-http-websocket-apis.md [hunk 1]
```diff
@@ -1,44 +1,35 @@
...
 cta_text: Get Started
 ---
-# Get Started with XRP Ledger APIs
+# Get Started Using HTTP / WebSocket APIs
 
-The XRP Ledger's core server software is [`rippled`](the-rippled-server.html). You can jump straight into developing on the XRP Ledger by accessing the API of a `rippled` server.
+If you don't have or don't want to use a [client library](client-libraries.html) in your preferred programming language, you can access the XRP Ledger directly through the APIs of its core server software, [`rippled`](the-rippled-server.html). The server provides APIs over JSON-RPC and WebSocket protocols. If you don't [run your own instance of `rippled`](install-rippled.html) you can still use a [public server][public servers].
 
-The quickest way to dive into the API is with the [**WebSocket API Tool**](websocket-api-tool.html), or use the [XRP Ledger Explorer](https://livenet.xrpl.org/) to watch the progress of the ledger live.
+**Tip:** You can dive right into the API is with the [**WebSocket API Tool**](websocket-api-tool.html), or use the [XRP Ledger Explorer](https://livenet.xrpl.org/) to watch the progress of the ledger live.
```
Comment (jbheron) [inline content/tutorials/get-started/get-started-using-http-websocket-apis.md]: Small typo, 'dive right into the API is with...'.

File: content/tutorials/get-started/get-started-using-http-websocket-apis.md [hunk 2]
```diff
@@ -50,7 +41,9 @@ Example WebSocket API request:
 }
 ```
 
-The response to [this command][server_info method] shows you the current status of the server. Or, read more about [Request Formatting](request-formatting.html) and [Response Formatting](response-formatting.html).
+The response to shows you the current status of the server.
```
Comment (jbheron) [inline content/tutorials/get-started/get-started-using-http-websocket-apis.md]: Missing noun here, 'The response to shows you...'.

File: content/tutorials/get-started/get-started-using-http-websocket-apis.md [hunk 3]
```diff
@@ -72,7 +65,9 @@ Content-Type: application/json
 }
 ```
 
-The response to this command shows you the current status of the server. For more information, see the [server_info method][].
+The response to shows you the current status of the server.
```
Comment (jbheron) [inline content/tutorials/get-started/get-started-using-http-websocket-apis.md]: Another 'The response to shows you...'.

File: content/tutorials/get-started/get-started-using-java.md [hunk 4]
```diff
@@ -38,10 +38,10 @@ In this tutorial, you'll learn:
...
 `xrpl4j` is split into multiple artifacts, which can be imported as needed.
 
-In this tutorial, you will need the [xrpl4j-client](https://javadoc.io/doc/org.xrpl/xrpl4j-client/latest/index.html), 
-[xrpl4j-address-codec](https://javadoc.io/doc/org.xrpl/xrpl4j-address-codec/latest/index.html), 
-[xrpl4j-keypairs](https://javadoc.io/doc/org.xrpl/xrpl4j-keypairs/latest/index.html), and 
-[xrpl4j-model](https://javadoc.io/doc/org.xrpl/xrpl4j-model/latest/index.html)  modules. 
+In this tutorial, you will need the [xrpl4j-client](https://javadoc.io/doc/org.xrpl/xrpl4j-client/latest/index.html),
+[xrpl4j-address-codec](https://javadoc.io/doc/org.xrpl/xrpl4j-address-codec/latest/index.html),
+[xrpl4j-keypairs](https://javadoc.io/doc/org.xrpl/xrpl4j-keypairs/latest/index.html), and
+[xrpl4j-model](https://javadoc.io/doc/org.xrpl/xrpl4j-model/latest/index.html)  modules.
```
Comment (jbheron) [inline content/tutorials/get-started/get-started-using-java.md]: Double whitespace on line 44.

**Review by jbheron — APPROVED** [review]
LGTM: I left a few comments about typos.

**Review by SirJosh1987 — COMMENTED** [review]
My bad

**Review by SirJosh1987 — COMMENTED** [review]
My bad

**Pull request merged**